doc/process/jenkins_job_configuation_access.md

```diff
@@ -0,0 +1,72 @@
+# Introduction
+
+There are a number of cases were we would like working group members
+to be able to run and or edit jobs within jenkins. Examples include
+the CITGM jobs and the Benchmarking jobs.
+
+By default jobs can only be edited by members of the
+'nodejs/jenkins-admins' group which contains a subset of the build
+workgroup members. We do not give access to
+everbody to run/edit as it would be a potential security issue,
+both through the potential disclosure of secrets used as
+part of the builds, as well as a potential denial
+of service if jobs are launched maliciously.
+
+This document outlines the processes by which we will allow
+people outside the build working group to edit jobs as well
+as allowing people who are not collaborators run specific jobs.
+
+Unfortunately jenkins does not allow us to easily delegate the
+creation of new jobs in an appropriate manner.
+As such, the creation of a new job will
+need to be requested through an issue in the build repo.  Any
+build member can then create a blank job in the group for the
```

Comment:
Maybe `create a blank job` -> `clone one of that workgroup's existing jobs`, as the cloning will then automatically pick up the right privileges.